app/frontend/styles/application.scss

```diff
@@ -37,5 +37,6 @@ $govuk-breakpoints: (
 @import "_qualification";
 @import "_section";
 @import "_status-box";
+@import "_status-indicator";
```

Comment:
I think this stylesheet can be scoped to just candidate, for now rather than be included for all interfaces.